specificatie/reisdocument.yaml

```diff
@@ -6,90 +6,104 @@ info:
       properties:
-        reisdocumentnummer:
-          $ref: '#/components/schemas/Reisdocumentnummer' 
-        indicatieGeldig:
+        datum:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+        aanduiding:
           $ref: 'common.yaml#/components/schemas/Waardetabel'
+        inOnderzoek:
+          $ref: '#/components/schemas/InhoudingOfVermissingInOnderzoek'
+    InhoudingOfVermissingInOnderzoek:
+      type: object
+      properties:
+        datum:
+          type: boolean
+        aanduiding:
+          type: boolean
+        datumIngangOnderzoek:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+    ReisdocumenthouderBasis:
+      type: object
+      properties:
         burgerservicenummer:
-          $ref: '#/components/schemas/Burgerservicenummer'
-    ReisdocumentUitgebreid:
+          $ref: 'persoon.yaml#/components/schemas/Burgerservicenummer'
+        opschortingBijhouding:
+          $ref: 'opschorting-bijhouding.yaml#/components/schemas/OpschortingBijhouding'
+    GbaReisdocumenthouder:
       allOf:
-        - $ref: '#/components/schemas/Reisdocument'
+        - $ref: '#/components/schemas/ReisdocumenthouderBasis'
         - type: object
-          description: |
-                        Een document dat nodig is voor reizen naar het buitenland.
-
-                        - indicatieGeldig geeft aan of het reisdocument op dit moment nog geldig en in het bezit van de persoon is. Wordt gevuld op basis van de waarden die voorkomen in de tabel 'Geldig_Reisdocument' uit de Haal-Centraal-BRP-tabellen-bevragen API.
-                        - soortReisdocument wordt gevuld op basis van de waarden die voorkomen in de tabel 'Nederlands_Reisdocument' uit de Haal-Centraal-BRP-tabellen-bevragen API.
           properties:
-            soortReisdocument:
-              $ref: 'common.yaml#/components/schemas/Waardetabel'
-            datumEindeGeldigheid:
-              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+            geheimhoudingPersoonsgegevens:
+              $ref: 'persoon.yaml#/components/schemas/GbaGeheimhoudingPersoonsgegevens'
+            inOnderzoek:
+              $ref: 'gba-inonderzoek.yaml#/components/schemas/GbaInOnderzoek'
+    Reisdocumenthouder:
+      allOf:
+        - $ref: '#/components/schemas/ReisdocumenthouderBasis'
+        - type: object
+          properties:
+            geheimhoudingPersoonsgegevens:
+              $ref: 'persoon.yaml#/components/schemas/GeheimhoudingPersoonsgegevens'
+            inOnderzoek:
+              $ref: '#/components/schemas/ReisdocumenthouderInOnderzoek'
+    ReisdocumenthouderInOnderzoek:
+      type: object
+      properties:
+        burgerservicenummer:
+          type: boolean
+        datumIngangOnderzoek:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+    Reisdocumentnummer:
+      type: string
+      pattern: ^[A-Z0-9]{9}$
```

Comment:
zie opmerking van Frans. Wellicht verstandig dit niet té scherp te definiëren, zit ook in de response, dus mogelijke waarden uitbreiden zou denk ik breaking zijn

Reply:
uitbreiden met\toevoegen van andere ranges zou niet breaking moeten zijn

Reply:
aangepast